website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Release Notes - Podman Desktop 1.2
+description: Podman Desktop 1.2 has been released!
+slug: podman-desktop-release-1.2
+authors: [cdrage]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.2 Release! 🎉
+
+We're excited to announce the release of Podman Desktop version 1.2.0! This release includes several new features, bug fixes, and improvements to enhance your container management experience. Here's a summary of the key changes in this release:
+
+- **Protocol handler support**: Added support for protocol handlers such as `open podman-desktop:extension/myextension`
+- **Kubernetes context on the status bar**: Kube context can now appear on the status bar
+- **Added start/stop/delete buttons for Compose**: You can now make changes to an entire Compose group of containers
+- **Ability to rename an image**: Rename an image with a click of a button
+
+Podman Desktop 1.2 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-1-2-hero](img/podman-desktop-release-1.2/podman-desktop-release-1.2.png)
+
+---
+
+## Release Details
+
+### Protocol handler support
+
+Podman Desktop now supports protocol handling when using the terminal! If you type in `open podman-desktop:extension/redhat.openshift-local`, Podman Desktop will automatically load up to the correct extension.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/243304511-b11ad1e4-4c2f-455c-957a-01653d2a93c8.mp4" />
+
+### Kubernetes context on the status bar
+
+On the status bar, the current Kubernetes context is now shown. If there are multiple contexts, you can now pick which one to use.
```

Comment:
How about "The current Kubernetes context is now shown on the status bar, making it easy to see the current context without leaving the window. If you want to change the context, just click to pick another context."